README.md

```diff
@@ -13,7 +13,7 @@ To address these emerging needs in the single cell ecosystem, the Chan Zuckerber
 
 1. Driving the development of SOMA.
 2. Providing its first implementation, [TileDB-SOMA](https://github.com/single-cell-data/TileDB-SOMA) which utilizes the [TileDB Embedded](https://github.com/TileDB-Inc/TileDB) engine.
-3. Adopting TileDB-SOMA at [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) to build the [Cell Census](https://github.com/chanzuckerberg/cell-census/) which provides efficient access and querying to a corpus containing nearly 50 million cells, compiled from 700+ datasets.
+3. Adopting TileDB-SOMA at [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) to build the [CELLxGENE Census](https://github.com/chanzuckerberg/cellxgene-census/) which provides efficient access and querying to a corpus containing nearly 50 million cells, compiled from 700+ datasets.
```

Comment:
With the context of CZ CELLxGENE Discover in place, we can refer to it as Census only. 

As a reminder the feature name is Census, the parent product is CZ CELLxGENE Discover

```suggestion
3. Adopting TileDB-SOMA at [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) to build its [Census](https://github.com/chanzuckerberg/cellxgene-census/) which provides efficient access and querying to a corpus containing nearly 50 million cells, compiled from 700+ datasets.
```